configure-system-libraries.sh

```diff
@@ -2,7 +2,7 @@
 # Patch *.dll.config files to use system libraries, working around issues with directories and naming schemes
 
 ####
-# This file must stay /bin/sh and POSIX compliant for macOS and BSD portability.
```

Comment:
This comment still applies for people who want to compile their own build using dotnet on BSD. These should IMO stay.